app/logic/AuditTrail.scala

```diff
@@ -7,7 +7,71 @@ import org.joda.time.{DateTime, DateTimeZone, Duration}
 import play.api.Logging
 import software.amazon.awssdk.services.dynamodb.model.AttributeValue
 
+import scala.util.Try
+
 object AuditTrail extends Logging {
+
+  val tableName = "AuditTrail"
+  val secondaryIndexName = "AuditTrailByUser"
+
+  val partitionKeyName = "j_account"
```

Comment:
This consolidation might also give us the opportunity to add a comment explaining the `j_` prefix, which is to avoid clashing with [DynamoDB reserved keywords](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ReservedWords.html). Up to you if you think this is relevant / useful?

Reply:
Add some comments about the database attributes in e291ff3dba9068780ab0183d218b9dc3c578576e